source/common/upstream/subset_lb.h

```diff
@@ -94,7 +94,9 @@ class SubsetLoadBalancer : public LoadBalancer, Logger::Loggable<Logger::Id::ups
 
   SubsetMetadata extractSubsetMetadata(const std::set<std::string>& subset_keys, const Host& host);
 
-  const HostSetImpl& emptyHostSet() { CONSTRUCT_ON_FIRST_USE(HostSetImpl); };
+  const HostSetImpl& emptyHostSet() {
+    CONSTRUCT_ON_FIRST_USE(HostSetImpl, original_host_set_.priority());
```

Comment:
Maybe add a comment stating that the host set priority will never change, hence why this is valid to do.

Reply:
Turns out this function isn't used